lib/realtime/api/tenant.ex

```diff
@@ -20,7 +20,7 @@ defmodule Realtime.Api.Tenant do
     field(:max_events_per_second, :integer, default: 100)
     field(:max_bytes_per_second, :integer, default: 100_000)
     field(:max_channels_per_client, :integer, default: 100)
-    field(:max_joins_per_second, :integer, default: 500)
+    field(:max_joins_per_second, :integer, default: 100)
```

Comment:
@chasers just a heads up i changed this to 100 to better reflect what the default limit should be.